zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -239,44 +241,13 @@ function PageWrapper(props) {
 
       <Container className={classes.childrenContainer} maxWidth="lg">
         {props.auth?.token ? <DashboardLayout>{loading ? <LoadingPage /> : props.children}</DashboardLayout> : null}
-        {!props.auth?.token &&
-          ![
-            '/',
-            '/signup',
-            '/login',
-            '/projects/:id',
-            '/ambassadors',
-            '/creators/:username',
-            '/privacy_policy',
-            '/terms_of_use',
-            '/about',
-            '/challenge',
-            '/password-reset',
-            '/email-confirm',
-            '/password-reset-confirm'
-          ].includes(props.match?.path) && (
-            <div style={{ minHeight: '80vh' }}>
-              <NotFoundPage />
-            </div>
-          )}
+        {!props.auth?.token && !props.global && (
```

Comment:
I'm a bit confused about this. Why did we change the previous implementation we had to this?
Also I looked but I can't see where global is from, so where are we getting `props.global` from?

(mentioning both @brrkrmn and @coderatomy so the person we introduced this change will get pinged by github)